### PR TITLE
Render numeric cell number for empty squares.

### PR DESCRIPTION
**Pull request by mkrump**
https://trello.com/c/5tqphubM/35-3-25-render-board-such-that-the-squares-are-labeled-to-match-console-input

### CODE REVIEW

File: TicTacToe/UIObejcts.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+            _colorVerticalCharacter = ConsoleColor.White;
+            _boardVerticalCharacter = "|";
+            _colorHorizontalCharacter = ConsoleColor.White;
+            _boardHorizontalCharacter = "-";
+            _spaceMarkerBackgroundColor = ConsoleColor.Black;
+            _spaceMarkerForegroundColor = ConsoleColor.White;
+            _console = console;
+        }
+
+
```
Comment (andarcabrera) [inline TicTacToe/UIObejcts.cs]: 👍 on keeping the below methods short and clean

File: TicTacToe/UIObejcts.cs [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+            _console.Write(_menuName);
+        }
+
+        public void RenderBoard(IBoard board)
+        {
+            for (int row = 0; row < board.GridSize; row++)
+            {
+                for (int column = 0; column < board.GridSize; column++)
+                {
+                    RenderSquareContents(board, row, column);
```
Comment (andarcabrera) [inline TicTacToe/UIObejcts.cs]: do you need to pass in the whole board object here?

**Comment by andarcabrera** [comment]
Are there tests for render_numeric board?

File: TicTacToe/UIObejcts.cs (hunk 2 above)
Comment (mkrump) [inline TicTacToe/UIObejcts.cs]: Probably not. I can just send in the parameters.

**Comment by mkrump** [comment]
re: tests for board rendering, like we talked about on call they are in UITests.cs.

**Pull request closed without merging**